aiohttp/web_urldispatcher.py

```diff
@@ -1121,3 +1115,18 @@ def add_routes(self,
         for route_def in routes:
             registered_routes.extend(route_def.register(self))
         return registered_routes
+
+
+def _quote_path(value: str) -> str:
+    return URL.build(path=value).raw_path
+
+
+def _unquote_path(value: str) -> str:
+    return URL.build(path=value, encoded=True).path
+
+
+def _requote_path(value: str) -> str:
```

Comment:
I'm struggling with the logic of this. Is this just reimplementing the yarl 1.5 behaviour when `URL.build(..., encoded=False)` is given an encoded string?

If so, it might be handy to document that, because it _looks_ like it quotes the path and then unquotes it for `%`, and even with that context, I'm having trouble parsing why we'd want to do that.